src/csi/sanity/sc.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 fsType: ext4
 transportType: "tcp"
-targetAddress: "107.108.83.97"
+targetAddress: "107.99.41.212"
 transportServiceId: "1158"
-provisionerIP: "107.108.83.97"
+provisionerIP: "107.99.41.212"
```

Comment:
please update it to localhost if possible

Reply:
Sure